<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.scripting.bsh.BshScriptFactoryTests" tests="18" skipped="0" failures="0" errors="0" timestamp="2020-12-16T17:33:54" hostname="DESKTOP-EMRC5FU" time="1.595">
  <properties/>
  <testcase name="scriptCompilationException" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.021"/>
  <testcase name="scriptThatCompilesButIsJustPlainBad" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.016"/>
  <testcase name="staticPrototypeScript" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.115"/>
  <testcase name="resourceScriptFromTag" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.151"/>
  <testcase name="prototypeScriptFromTag" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.143"/>
  <testcase name="staticScriptWithNullReturnValue" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.134"/>
  <testcase name="staticWithScriptReturningInstance" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.14"/>
  <testcase name="staticScriptWithTwoInterfacesSpecified" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.136"/>
  <testcase name="staticScriptImplementingInterface" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.135"/>
  <testcase name="inlineScriptFromTag" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.127"/>
  <testcase name="ctorWithWhitespacedScriptSourceLocator" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.0"/>
  <testcase name="ctorWithNullScriptSourceLocator" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.0"/>
  <testcase name="applicationEventListener" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.124"/>
  <testcase name="staticScript" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.142"/>
  <testcase name="refreshableFromTag" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.133"/>
  <testcase name="ctorWithEmptyScriptSourceLocator" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.0"/>
  <testcase name="nonStaticScript" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.041"/>
  <testcase name="nonStaticPrototypeScript" classname="org.springframework.scripting.bsh.BshScriptFactoryTests" time="0.036"/>
  <system-out><![CDATA[01:33:55.007 [Test worker] WARN  o.s.c.s.ClassPathXmlApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'broken' defined in class path resource [org/springframework/scripting/bsh/bshBrokenContext.xml]: BeanPostProcessor before instantiation of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'scriptedObject.broken' defined in class path resource [org/springframework/scripting/bsh/bshBrokenContext.xml]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [java.lang.Object]: Factory method 'getScriptedObject' threw exception; nested exception is org.springframework.scripting.ScriptCompilationException: Could not compile class path resource [org/springframework/scripting/bsh/Broken.bsh]; nested exception is In file: inline evaluation of: ``one sure is the loneliest number... that i'll ever know;'' Encountered "is" at line 1, column 10.

org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@c6665d, started on Thu Dec 17 01:33:55 CST 2020]
org.springframework.context.event.ContextClosedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@c6665d, started on Thu Dec 17 01:33:55 CST 2020]
org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@b3c36c, started on Thu Dec 17 01:33:55 CST 2020]
org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@ce6052, started on Thu Dec 17 01:33:55 CST 2020]
org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@967797, started on Thu Dec 17 01:33:56 CST 2020]
org.springframework.scripting.bsh.BshScriptFactoryTests$MyEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@967797, started on Thu Dec 17 01:33:56 CST 2020]
org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@14b48c9, started on Thu Dec 17 01:33:56 CST 2020]
]]></system-out>
  <system-err><![CDATA[Dez 17, 2020 1:33:55 AM ClientIntermediary close
INFORMATION: java.io.IOException: The connection is not currently established.
java.io.IOException: The connection is not currently established.
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl.checkState(ClientSynchroMessageConnectionImpl.java:617)
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl.sendOneWay(ClientSynchroMessageConnectionImpl.java:202)
	at javax.management.remote.generic.GenericConnector.close(GenericConnector.java:300)
	at javax.management.remote.generic.GenericConnector.close(GenericConnector.java:271)
	at javax.management.remote.generic.ClientIntermediary$GenericClientCommunicatorAdmin.doStop(ClientIntermediary.java:897)
	at com.sun.jmx.remote.opt.internal.ClientCommunicatorAdmin.restart(ClientCommunicatorAdmin.java:172)
	at com.sun.jmx.remote.opt.internal.ClientCommunicatorAdmin.gotIOException(ClientCommunicatorAdmin.java:73)
	at javax.management.remote.generic.GenericConnector$RequestHandler.execute(GenericConnector.java:662)
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl$RemoteJob.run(ClientSynchroMessageConnectionImpl.java:568)
	at com.sun.jmx.remote.opt.util.ThreadService$ThreadServiceJob.run(ThreadService.java:248)
	at com.sun.jmx.remote.opt.util.JobExecutor.run(JobExecutor.java:99)

Dez 17, 2020 1:33:55 AM ClientCommunicatorAdmin restart
WARNUNG: Failed to restart: java.net.ConnectException: Connection refused: connect
Dez 17, 2020 1:33:55 AM ClientCommunicatorAdmin restart
WARNUNG: Failed to restart: java.net.ConnectException: Connection refused: connect
Dez 17, 2020 1:33:56 AM ClientIntermediary close
INFORMATION: java.io.IOException: The connection is not currently established.
java.io.IOException: The connection is not currently established.
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl.checkState(ClientSynchroMessageConnectionImpl.java:617)
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl.sendOneWay(ClientSynchroMessageConnectionImpl.java:202)
	at javax.management.remote.generic.GenericConnector.close(GenericConnector.java:300)
	at javax.management.remote.generic.GenericConnector.close(GenericConnector.java:271)
	at javax.management.remote.generic.ClientIntermediary$GenericClientCommunicatorAdmin.doStop(ClientIntermediary.java:897)
	at com.sun.jmx.remote.opt.internal.ClientCommunicatorAdmin.restart(ClientCommunicatorAdmin.java:172)
	at com.sun.jmx.remote.opt.internal.ClientCommunicatorAdmin.gotIOException(ClientCommunicatorAdmin.java:73)
	at javax.management.remote.generic.GenericConnector$RequestHandler.execute(GenericConnector.java:662)
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl$RemoteJob.run(ClientSynchroMessageConnectionImpl.java:568)
	at com.sun.jmx.remote.opt.util.ThreadService$ThreadServiceJob.run(ThreadService.java:248)
	at com.sun.jmx.remote.opt.util.JobExecutor.run(JobExecutor.java:99)

Dez 17, 2020 1:33:56 AM ClientIntermediary close
INFORMATION: java.io.IOException: The connection is not currently established.
java.io.IOException: The connection is not currently established.
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl.checkState(ClientSynchroMessageConnectionImpl.java:617)
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl.sendOneWay(ClientSynchroMessageConnectionImpl.java:202)
	at javax.management.remote.generic.GenericConnector.close(GenericConnector.java:300)
	at javax.management.remote.generic.GenericConnector.close(GenericConnector.java:271)
	at javax.management.remote.generic.ClientIntermediary$GenericClientCommunicatorAdmin.doStop(ClientIntermediary.java:897)
	at com.sun.jmx.remote.opt.internal.ClientCommunicatorAdmin.restart(ClientCommunicatorAdmin.java:172)
	at com.sun.jmx.remote.opt.internal.ClientCommunicatorAdmin.gotIOException(ClientCommunicatorAdmin.java:73)
	at javax.management.remote.generic.GenericConnector$RequestHandler.execute(GenericConnector.java:662)
	at com.sun.jmx.remote.generic.ClientSynchroMessageConnectionImpl$RemoteJob.run(ClientSynchroMessageConnectionImpl.java:568)
	at com.sun.jmx.remote.opt.util.ThreadService$ThreadServiceJob.run(ThreadService.java:248)
	at com.sun.jmx.remote.opt.util.JobExecutor.run(JobExecutor.java:99)

]]></system-err>
</testsuite>
